app/utils/cache.server.ts

```diff
@@ -161,29 +162,12 @@ export async function cachified<Value>({
 	request?: Request
 	timings?: Timings
 }): Promise<Value> {
-	let cachifiedResolved = false
-	const cachifiedPromise = baseCachified({
-		reporter: verboseReporter(),
+	return baseCachified({
 		...options,
-		getFreshValue: async context => {
-			// if we've already retrieved the cached value, then this may be called
-			// after the response has already been sent so there's no point in timing
-			// how long this is going to take
-			if (!cachifiedResolved && timings) {
-				return time(() => options.getFreshValue(context), {
-					timings,
-					type: `getFreshValue:${options.key}`,
-					desc: `request forced to wait for a fresh ${options.key} value`,
-				})
-			}
-			return options.getFreshValue(context)
-		},
-	})
-	const result = await time(cachifiedPromise, {
-		timings,
-		type: `cache:${options.key}`,
-		desc: `${options.key} cache retrieval`,
+		reporter: mergeReporters(
+			cachifiedTimingReporter(timings),
+			options.reporter,
+			verboseReporter(),
+		),
```

Comment:
This makes it impossible to override (disable) the `verboseReporter` as it was before. Can we refactor it a bit to make that possible? Will this work?

```suggestion
		reporter: mergeReporters(
			cachifiedTimingReporter(timings),
			options.reporter ?? verboseReporter(),
		),
```